packages/zowe-explorer/__tests__/__unit__/ZoweNode.unit.test.ts

```diff
@@ -78,139 +79,139 @@ describe("Unit Tests (Jest)", () => {
     /*************************************************************************************************************
      * Creates sample ZoweDatasetNode list and checks that getChildren() returns the correct array
      *************************************************************************************************************/
-    it("Testing that getChildren returns the correct Thenable<ZoweDatasetNode[]>", async () => {
-        Object.defineProperty(Profiles, "getInstance", {
-            value: jest.fn(() => {
-                return {
-                    loadNamedProfile: jest.fn().mockReturnValue(profileOne),
-                };
-            }),
-        });
-        // Creating a rootNode
-        const rootNode = new ZoweDatasetNode({
-            label: "root",
-            collapsibleState: vscode.TreeItemCollapsibleState.Collapsed,
-            session,
-            profile: profileOne,
-        });
-        rootNode.dirty = true;
-        rootNode.contextValue = globals.DS_SESSION_CONTEXT;
-        rootNode.pattern = "SAMPLE, SAMPLE.PUBLIC, SAMPLE";
-        let rootChildren = await rootNode.getChildren();
+    // it("Testing that getChildren returns the correct Thenable<ZoweDatasetNode[]>", async () => {
```

Comment:
Do we plan to implement these tests again?

I may run into these commented out tests in some other form later during the review process, but just wanted to raise the question in case we just forgot about them 😋

Reply:
These Data Set tests also had issues with circular structures. I don't mind writing new tests for this function though.

The USS node has coverage because we now use a function in the provider to build the children nodes. I will look into a similar implementation for Data Set nodes.

Reply:
I think you've done a pretty good job at keeping up with code-coverage.
Feel free to remove any commented out tests once you are done 😋